phpstan.neon.dist

```diff
@@ -9,4 +9,4 @@ parameters:
     level: 8
     featureToggles:
         alwaysTrueAlwaysReported: true
-        instanceofType: true
```

Comment:
👀

Reply:
yeah, sorry. yesterday was a bad day :D

Reply:
I actually didn't enable the list type in that other PR